### DISCUSSION TITLE
Confusion about running via docker with env.yaml on host

### DISCUSSION THREAD

**Discussion by jbwl** [slingdata-io/sling-cli#401] [General]
Please excuse me if this is a non-issue - maybe what I'm trying to do is not possible when sling is run via docker?

## Issue Description

- Description of the issue: 
I have an env.yaml file holding multiple connections in my ~/.sling path on the host. This file is valid because running sling without docker detects all the connections in it.

But when I try to run sling via docker with this command:
`docker run -e SLING_HOME_DIR=/home/me/.sling/ --rm -i slingdata/sling conns list`
then only this connection is listed:
```
+-----------+-----------------+----------+
| CONN NAME | CONN TYPE       | SOURCE   |
+-----------+-----------------+----------+
| LOCAL     | FileSys - Local | built-in |
+-----------+-----------------+----------+
```

- Sling version (`sling --version`): 
1.2.20

- Operating System (`linux`, `mac`, `windows`): 
Linux (Ubuntu 22.04)

- Replication Configuration: 
`docker pull slingdata/sling`
..and run the docker command above

- Log Output (please run command with `-d`): 
-d outputs help text, no log

**Comment by flarco** [comment]
Hi, you need to mount on docker volume:

```
docker run -v /home/me/.sling:/home/sling/.sling --rm -i slingdata/sling conns list
```

**Reply by jbwl** [reply]
Thank you, that works.
